config/prod.exs

```diff
@@ -12,7 +12,12 @@ import Config
 config :screens, ScreensWeb.Endpoint,
   url: [host: "example.com", port: 80],
   server: true,
-  cache_static_manifest: "priv/static/cache_manifest.json"
+  cache_static_manifest: "priv/static/cache_manifest.json",
+  check_origin: [
+    "https://screens.mbta.com",
+    "https://screens-dev.mbtace.com",
+    "https://screens-dev-green.mbtace.com"
+  ]
```

Comment:
Slightly nervous about this setting. I looked it up in the docs, and it seems to only apply to websocket connections, but if it messes with CORS settings for regular requests, that will cause problems for the DUP and triptych clients. Just something to keep an eye out for when this goes to prod.

Reply:
I should have mentioned this in the description, but I did make sure to test a DUP package before posting the PR. I didn't see anything off, even with a fake mraid object.